### ISSUE TITLE
Dropdown GUI freezes and fails

### ISSUE THREAD

**Issue by Phosu-Lite** [godotengine/godot#63729]
### Godot version

4.0 alpha13

### System information

Windows 8.1

### Issue description

Dropdown menus, both in the project selector and after loading a project, freeze the window for a couple seconds and fail to actually present the dropdown in latest alpha, but work fine on the same machine in stable.

### Steps to reproduce

No example project because it happens before even opening a project. Attempting to select the "sort by [date/name/etc]" dropdown freezes the window for a second or two, but when it revives I never actually see the dropdown. All in-project dropdowns I tried behaved like this as well. However, attempting to change the language also freezes for a couple seconds, but successfully loads the language dropdown behind the main window; I can still select languages normally where they stick out. It's the only dropdown I've found in or out of a project that works at all in alpha13. Multiple attempts don't fix the issue, nor does restarting so it's not https://github.com/godotengine/godot/issues/61758. It's also not https://github.com/godotengine/godot/issues/61773 as far as I understand that issue, as all other use of the editor is functional, and alt-tabbing doesn't affect the symptoms. 

3.4.4 stable works just fine, so I tried 4.0 alpha1. In alpha1, dropdowns still cause hangs, but all dropdowns except language work normally after the hang. After clicking exactly once, without ever showing a dropdown, after the hang it changes my language and a popup asks if I'd like to restart. It seems to be whichever one would be behind where I clicked on the dropdown UI, despite only clicking once--consistently the first is Bengali, then as I keep choosing "continue" and changing language again, it went down to Czech, then Catalan, and it looked alphabetical so I stopped.

So it broke somewhere between 3.4 and 4.0, and broke a little more somewhere between alpha1 and alpha9 (first alpha I tried to use, same symptoms on both alpha12 and alpha13). I can try testing the 7 alphas between them if necessary, but I want to just get this bug report out for now.

**Comment by Calinou** [comment]
Can you reproduce this after enabling Single Window Mode in the Editor Settings (or starting the project manager with the `--single-window` command line argument)?

**Comment by Phosu-Lite** [comment]
Nope, that fixed everything. Seems to run smoother in general now too.
